_data/projects/Aidre.yml

```diff
@@ -1,5 +1,7 @@
+---
 name: Aidre
-desc: 🐍 📝 Python Translation Help Station - The software for 📄 .po file docs handling. A tool to help with official py docs translations
+desc: "\U0001F40D \U0001F4DD Python Translation Help Station - The software for \U0001F4C4 .po file docs
```

Comment:
Emitting these YAML strings converted most of the emoji to their Unicode representation. Will confirm the client-side renders these fine on the staged deploy.

Reply:
Working as expected on the client:

<img width="1118" src="https://user-images.githubusercontent.com/359239/64168602-48606e80-ce22-11e9-8d04-e4b5101c5b64.png">